test/integration_test/testspec/migration/daemonset-with-all-components.yaml

```diff
@@ -132,7 +132,7 @@ spec:
             - name: dbusmount
               mountPath: /var/run/dbus
         - name: csi-node-driver-registrar
-          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
```

Comment:
do we update the default versions for daemonset installation? if not probably we don't need to change here.

Reply:
Yes we updated the version for the daemonset version